### PR TITLE
Unpick lightmap constants.

### PR DESCRIPTION
**Pull request by LambdAurora**
With some bonus method mappings that I encountered while unpicking.

### CODE REVIEW

File: unpick-definitions/lightmap_coordinates.unpick [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+target_method net/minecraft/client/render/entity/EntityRenderer getLight (Lnet/minecraft/entity/Entity;F)I
+	return lightmap_coordinates
+target_method net/minecraft/client/render/entity/EntityRenderer renderLabelIfPresent (Lnet/minecraft/entity/Entity;Lnet/minecraft/text/Text;Lnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V
+	param 4 lightmap_coordinates
+target_method net/minecraft/client/render/entity/EntityRenderer render (Lnet/minecraft/entity/Entity;FFLnet/minecraft/client/util/math/MatrixStack;Lnet/minecraft/client/render/VertexConsumerProvider;I)V
+	param 5 lightmap_coordinates
+
+target_method net/minecraft/client/render/entity/ExperienceOrbEntityRenderer vertex (Lnet/minecraft/client/render/VertexConsumer;Lnet/minecraft/util/math/Matrix4f;Lnet/minecraft/util/math/Matrix3f;FFIIIFFI)V
+	param 9 lightmap_coordinates
```
Comment (liach) [inline unpick-definitions/lightmap_coordinates.unpick]:
```suggestion
	param 10 lightmap_coordinates
```

**Review by liach — COMMENTED** [review]
Cannot find all the typos, but I think a few typos won't cause major troubles

File: unpick-definitions/lightmap_coordinates.unpick (hunk 1 above)
Comment (LambdAurora) [reply unpick-definitions/lightmap_coordinates.unpick]: The arguments numbering is quite confusing, once it starts at 0, once it starts at 1, it's not consistent at all in Enigma mappings and it leads to quite the confusion.

Comment (liach) [reply unpick-definitions/lightmap_coordinates.unpick]: In unpick format, it is the index of the argument in the argument array in the descriptor than the actual slot of the variable, I think, so you don't apply an offset for instance methods. Forgot how it treats long/double though.

**Review by liach — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**